src/components/navigation/footer-navigation/footer-navigation.js

```diff
@@ -97,7 +97,12 @@ export default class Navigation extends React.PureComponent {
           />
         </span>
       ) : link.external ? (
-        <ExternalLink link={link} key={i} />
+        <ExternalLink
+          link={link}
+          key={i}
+          target="_blank"
+          rel="noopener noreferrer"
```

Comment:
This should be inside the external-link-box.js. Also, considering that it's an external link probably it should include a nofollow